CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.1] - 2019-07-03
+
+### Fixed
+- Handle `null` component `ref`s in React Navigation HOC.
```

Comment:
While I think this is meaningful to us, I'd like to see something more descriptive for the external consumer.  Perhaps: 
```
Fixed a bug with React Navigation autocapture, where wrapping a navigator in a custom Higher-Order Component could prevent navigation events from being captured.
```